mappings/net/minecraft/block/AbstractBlock.mapping

```diff
@@ -91,7 +91,8 @@ CLASS net/minecraft/class_4970 net/minecraft/block/AbstractBlock
 	METHOD method_9535 getRenderingSeed (Lnet/minecraft/class_2680;Lnet/minecraft/class_2338;)J
 		ARG 1 state
 		ARG 2 pos
-	METHOD method_9536 onBlockRemoved (Lnet/minecraft/class_2680;Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_2680;Z)V
+	METHOD method_9536 onStateReplaced (Lnet/minecraft/class_2680;Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_2680;Z)V
+		COMMENT Called in {@link World#setBlockState()} if {@code newState} is different from {@code state}. Vanilla blocks perform removal cleanups here.
```

Comment:
```suggestion
		COMMENT Called in {@link net.minecraft.world.ModifiableWorld#setBlockState(BlockPos, BlockState, int)} if {@code newState} is different from {@code state}. Vanilla blocks perform removal cleanups here.
```
Sorry, my bad! https://travis-ci.com/github/FabricMC/yarn/jobs/336377975#L365
Those classes should've been imported in block class' method descriptors, so guess don't need fully qualified names